src/Configuration.php

```diff
@@ -85,4 +98,36 @@ public function valueIsTrue($key)
     {
         return (bool) $this->configuration[$key];
     }
+
```

Comment:
I suggest to remove `\ArrayAccess`, will be better to protect any magic in Analyzer
Lets use `$this->configuration`